builderstest/chaopod.go

```diff
@@ -0,0 +1,338 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
This file should probably be `chaospod`.go?

Reply:
🙈 Typo error...